app/models/lookup_value.rb

```diff
@@ -121,4 +121,8 @@ def ensure_matcher_exists
   def skip_strip_attrs
     ['value']
   end
+
+  def found_host?(fqdn)
```

Comment:
Would maybe `host_with_fqdn_exists?` describe bit better what the method does?

Reply:
Done

Reply:
Just to mention that a customer applied this patch and confirmed it is working as expected. You can check the bugzilla linked to this issue for more detail.